src/c++/perf_analyzer/command_line_parser.h

```diff
@@ -145,6 +145,9 @@ struct PerfAnalyzerParameters {
 
   // Triton inference response output tensor format.
   cb::TensorFormat output_tensor_format{cb::TensorFormat::BINARY};
+
+  // The profile export file path.
+  std::string profile_export_file{""};
```

Comment:
Should we make this default to the current working directory "."?

Reply:
It's supposed to hold a file path, so directory won't make sense.

Reply:
Don't we need empty string to signal that no file should be output? That's how the csv output arg is handled I believe.

Reply:
discussed offline. I misunderstood this partially and i think its fine now.